scripts/netmaker.default.env

```diff
@@ -86,4 +86,8 @@ EMAIL_SENDER_ADDR=
 EMAIL_SENDER_USER=
 # sender smtp password
 EMAIL_SENDER_PASSWORD=
+# if batch peer update enable or not
+PEER_UPDATE_BATCH=true
+# batch peer update size when PEER_UPDATE_BATCH is enabled
+PEER_UPDATE_BATCH_SIZE=50
```

Comment:
can you submit a separate PR for this to the v0.25.0 release branch?

Reply:
yes, another PR https://github.com/gravitl/netmaker/pull/3101 to v0.25.0